core/src/main/java/org/pentaho/cdf/storage/StorageEngine.java

```diff
@@ -56,14 +57,22 @@ public static Operation get( String operation ) {
     }
   }
 
-  public static synchronized StorageEngine getInstance() {
+  public static synchronized StorageEngineInterface getInstance() {
```

Comment:
this method is called from classes in other packages:
https://github.com/webdetails/cdf/blob/master/pentaho/src/main/java/org/pentaho/cdf/context/ContextEngine.java#L397
https://github.com/mtseu/cdf/blob/master/pentaho/src/main/java/org/pentaho/cdf/render/CdfHtmlRenderer.java#L364